mozilla/perfcompare
# Set High Confidence as Default Filter in Results Table

This Pr Resolves this issue - **[#766](https://github.com/mozilla/perfcompare/issues/766)**

**Default Filter Settings**: Updated the filtering logic to set "High" as the default selected confidence level. This change ensures that when the results table is loaded, it will only display high-confidence results first, aligning with user expectations.
  
**Filter Toggle Logic**: Adjusted the logic for toggling filters to ensure that only the selected confidence levels are displayed. If "High" is selected, only high-confidence results will be visible initially, while "Medium" and "Low" results will remain hidden until explicitly selected by the user.

#### Testing Steps
1. Navigate to the results table in the application.
2. Verify that high-confidence results are displayed by default upon loading the table.
3. Test the filter functionality by selecting "Medium" and "Low" to ensure that only the corresponding results are shown.
4. Confirm that re-selecting "High" brings back the high-confidence results.

#### Expected Outcome
With these changes, users should now experience a more intuitive results view, with high-confidence results prioritized, thus improving the overall usability of the application.

### Demo:
![Screenshot - 2024-10-11T035630 855](https://github.com/user-attachments/assets/1b47ab57-1835-49a6-a033-13759f183ff8)
